debian/rules

```diff
@@ -24,7 +24,8 @@ export DEB_LDFLAGS_MAINT_APPEND := -lpthread
 override_dh_auto_configure:
 	dh_auto_configure -B$(CURDIR)/build -- \
 	-DBUILD_VERSION=$(DEB_VERSION_UPSTREAM) \
-	-DBSONCXX_POLY_USE_BOOST=1
+	-DBSONCXX_POLY_USE_BOOST=1 \
+	-DENABLE_UNINSTALL=OFF
```

Comment:
Was disabling uninstall necessary?

Reply:
Yes.  Otherwise, we generate and install a file (`uninstall.sh`) which the packages don't install.  The build scripts will notice that a file was left behind and then fail the package build.